plugins/plugin-languageserver/che-plugin-languageserver-ide/src/main/java/org/eclipse/che/plugin/languageserver/ide/service/PublishDiagnosticsReceiver.java

```diff
@@ -10,17 +10,36 @@
  */
 package org.eclipse.che.plugin.languageserver.ide.service;
 
-import com.google.inject.Inject;
-import com.google.inject.Provider;
-import com.google.inject.Singleton;
+import com.google.web.bindery.event.shared.EventBus;
+import javax.inject.Inject;
```

Comment:
What's the policy around javax.inject vs. com.google.inject?

Reply:
javax.inject is more preferable than com.google.inject